core/src/main/java/org/bitcoinj/core/BloomFilter.java

```diff
@@ -167,6 +167,15 @@ public String toString() {
         return helper.toString();
     }
 
+    @Override
+    public int messageSize() {
+        int size = VarInt.sizeOf(data.length) + data.length;
+        size += 4; // hashFuncs
+        size += 4; // nTweak
+        size += 1; // nFlags
+        return size;
+    }
```

Comment:
I would put the `return` on the first line of the function, but still keep the commented lines, but just as:

```
  + 4   // hashFuncs
```

Reply:
Done.